pkg/policy/api/selector.go

```diff
@@ -21,18 +21,13 @@ import (
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/logfields"
 
+	"fmt"
```

Comment:
It's seems at least this commit was created for this branch, you should merge this commit first on master and then back port it to `v0.12`

Reply:
I cannot merge this into master. The new logging changes done by Ray in master are not to be back-ported and this was a way to get around the merge conflicts that created.

Reply:
My comment was for the whole commit, I was also referring on the method's descriptions changes bellow.